tests/test.py

```diff
@@ -118,6 +116,16 @@ def testNeuronsHaveTypes(self):
         # NOTE: Neurons ALNL, CANL, CANR, ALNR have unknown function and type
         self.assertEqual(len(results), len(self.neurons) - 4, "Some neurons are missing a type: {}".format(set(self.neurons) - results))
 
+    def test_neuron_GJ_degree(self):
+        """ Get the number of gap junctions from a networkx representation """
+        self.assertEqual(PyOpenWorm.Neuron(name='AVAL').GJ_degree(), 40)
+
+    def test_neuron_Syn_degree(self):
+        """ Get the number of chemical synapses from a networkx representation """
+        # XXX: This result is 1 greater compared to what would be expected from the
+        # connectome.csv file
+        self.assertEqual(PyOpenWorm.Neuron(name='AVAL').Syn_degree(), 91)
```

Comment:
For both questions, those are the results from counting what's in the RDF database. NetworkX has a strange way of counting the in-degree and out-degree. The gap junction counts, for instance, don't match for in- and out-degree. Coming from the CSV file that the NetworkX graph is generated from, the counts from the database are more accurate. I looked at the edges that are left out for GapJunction counts and there's not an immediately obvious pattern. I haven't looked yet at which of the connections makes the difference in Syn_degree since I'm anticipating #89, which has our highest integrity source of synapse info. Having a test that matches the database lets us know if there is a difference, whichever PR is merged first--that's also why I moved these tests into the DataIntegrityTest class since they're based on the RDF data now.